### ISSUE TITLE
Has anybody succeeded in installing golang on gnuroot debian?

### ISSUE THREAD

**Issue by mygithubcomments2017** [corbinlc/GNURootDebian#218]
I cross-posted a related question on golang foum: https://forum.golangbridge.org/t/i-cant-compile-hello-world-go-program-on-android-4-2-gnuroot-debian/6945?u=mygithubcomments2017

**Comment by cdrasmussen** [comment]
As a test, I just installed golang without a hitch. It ran help and version
without a problem as well. I have not tried to build hello world though.

This was on an LG Stylo 3 if that helps.

+CDR

**Comment by mygithubcomments2017** [comment]
@cdrasmussen: Yes help and version works for me too, I just can't compile and run anything including the 'Hello, world' sample from here: https://golang.org/doc/install --> “Test your installation”

**Comment by cdrasmussen** [comment]
I followed your suggestion and while it would build the binary would segv.
I tried different setting per https://github.com/golang/go/wiki/GoArm but
it didn't help. The go env cmd wants no part of armhf which is what the
kernel is.


+CDR

**Comment by AdamMatthewLTM** [comment]
Hi all,

I am currently assisting the developers in sorting relevant issues for GNURoot Debian on GitHub. If the issue discussed is still relevant, could you please reply to this thread and state the issue needs to be resolved and I will attempt to sort it for you ASAP. If the issue is no longer relevant, please reply to this thread and state it has been resolved, and I will suggest this thread be closed.

Furthermore, if a particular issue is considered to be high priority, please state that you are requesting it to be considered as high priority and I will mark it as such.

Thank you all so much for your time and understanding,
-Adam
